sdk/spring/azure-spring-boot-starter-active-directory/README.md

```diff
@@ -211,10 +135,64 @@ azure:
         scopes: {scope1}, {scope2}
 ```
 
-### [Web API] Protect the resource APIs in resource server
-Please refer to [azure-spring-boot-sample-active-directory-resource-server] for access resource APIs.
+#### Related Java code
+1. Resource servers should be registered as `ClientRegistration`.
+2. In Controller, `@RegisteredOAuth2AuthorizedClient` can be used to  get `OAuth2AuthorizedClient`. 
+   `OAuth2AuthorizedClient` contains accessToken to access related client resource.
 
-#### Using `AADOAuth2ResourceServerSecurityConfig` to extends `WebSecurityConfigurerAdapter`:
+Support the use of `@RegisteredOAuth2AuthorizedClient` to get `OAuth2AuthorizedClient`:
+```java
+@GetMapping("/graph")
+@ResponseBody
+public String graph(@RegisteredOAuth2AuthorizedClient("graph") OAuth2AuthorizedClient oAuth2AuthorizedClient) {
+    return toJsonString(oAuth2AuthorizedClient);
+}
+
+@GetMapping("/office")
+@ResponseBody
+public String office(@RegisteredOAuth2AuthorizedClient("office") OAuth2AuthorizedClient oAuth2AuthorizedClient) {
+    return toJsonString(oAuth2AuthorizedClient);
+}
+```
+
+### Resource Server
+Based on Azure AD as a Resource Server, it uses `BearerTokenAuthenticationFilter` authorize request. The current resource server also can access other resources, there's a similar method to the web application usage to obtain access to the client access token, the difference is the access token obtained based on the `MSAL On-Behalf-Of` process.
```

Comment:
`The current resource server also can access other resources..`, this should move to `Resource server access other resources`